.github/workflows/test.yml

```diff
@@ -39,9 +39,7 @@ jobs:
 
       - name: Test core with pytest
         run: |
-          pytest src/fairchem/data/oc/tests -vv --cov-report=xml --cov=fairchem.data.oc
-          pytest src/fairchem/demo/ocpapi/tests -vv --cov-report=xml --cov=fairchem.demo.ocpapi
-          pytest src/fairchem/core/tests -vv --cov-report=xml --cov=fairchem.core
+          pytest tests -vv --cov-report=xml --cov=fairchem --collect-only -c ./packages/fairchem-core/pyproject.toml
```

Comment:
--collect-only seems to only collect tests but not run them

Reply:
ah good catch! I was debugging and left it in there! Thank you!